vlib/rand/README.md

```diff
@@ -38,33 +65,14 @@ This is often useful for simulations that need the same starting seed.
 If you need truly random numbers that are going to be used for cryptography,
 use the `crypto.rand` module.
 
-# Guaranteed functions
-
-The following 21 functions are guaranteed to be supported by `rand`
-as well as the individual PRNGs.
-
-- `seed(seed_data)` where `seed_data` is an array of `u32` values.
-    Different generators require different number of bits as the initial seed.
-    The smallest is 32-bits, required by `sys.SysRNG`.
-    Most others require 64-bits or 2 `u32` values.
-- `u32()`, `u64()`, `int()`, `i64()`, `f32()`, `f64()`
-- `u32n(max)`, `u64n(max)`, `intn(max)`, `i64n(max)`, `f32n(max)`, `f64n(max)`
-- `u32_in_range(min, max)`, `u64_in_range(min, max)`, `int_in_range(min, max)`,
-    `i64_in_range(min, max)`, `f32_in_range(min, max)`, `f64_in_range(min, max)`
-- `int31()`, `int63()`
-
-There are several additional functions defined in the top-level module that rely
-on the global RNG. If you want to make use of those functions with a different
-PRNG, you can can change the global RNG to do so.
```

Comment:
Why delete this?

Reply:
Because we now have (almost) 1-1 function parity b/w the PRNG interface and the top-level functions.